src/styles/block/card.scss

```diff
@@ -0,0 +1,118 @@
+$fontColorBlue: #060b35;
+$fontColorGray: #616070;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  align-items: center;
+
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  background: #fff;
```

Comment:
You can use this color as a variable, because this color is repeated several times in the code